sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/QueueProperties.java

```diff
@@ -132,6 +134,7 @@ public void setName(QueueProperties queueProperties, String name) {
         this.forwardDeadLetteredMessagesTo = description.getForwardDeadLetteredMessagesTo();
         this.lockDuration = description.getLockDuration();
         this.maxDeliveryCount = toPrimitive(description.getMaxDeliveryCount());
+        this.maxMessageSizeInKilobytes = toPrimitive(description.getMaxMessageSizeInKilobytes());
```

Comment:
Above Ramya mentioned the service may return `null` for this property when targetting older API versions. If that's the case, then we're converting that `null` to`0L` here, if so, when we send the same model payload to service for an update, we will be sending `0L`, which might cause the service to throw an error.

Reply:
Yeah Anu, as long as we don't send the parameter when it is 0L we should be good here. And that is what we do, before doing a create / update we only include this parameter if it is non-zero